integration_test/cases/repo.exs

```diff
@@ -763,8 +762,25 @@ defmodule Ecto.Integration.RepoTest do
     # With order_by and limit
     query = from Post, order_by: [asc: :visits], limit: 2
     assert TestRepo.aggregate(query, :max, :visits) == 12
+  end
+
+  @tag :average_with_precision
```

Comment:
Following our discussion, this can probably be called `@tag :decimal_precision` too!

Reply:
renamed